src/routes/pricing/+page.svelte

```diff
@@ -5,13 +5,40 @@
     import { Main } from '$lib/layouts';
     import { TITLE_SUFFIX } from '$routes/titles';
     import ComparePlans from './compare-plans.svelte';
+    import CompareSupport from './compare-support.svelte';
     import Faq from './faq.svelte';
     import BG from './bg.png?enhanced';
     import { PUBLIC_APPWRITE_DASHBOARD } from '$env/static/public';
+    import Toggle from '$lib/components/ui/Toggle.svelte';
+    import { goto } from '$app/navigation';
+    import { onMount } from 'svelte';
 
     const title = 'Pricing' + TITLE_SUFFIX;
     const description = 'Explore our straightforward pricing plans that scale with your project.';
     const ogImage = DEFAULT_HOST + '/images/open-graph/website.png';
+
+    const pageStateOptions: Array<{ label: string; key: string }> = [
+        { label: 'Cloud', key: 'cloud' },
+        { label: 'Self hosted', key: 'self-hosted' }
+    ];
+
+    let activeToggle: string = 'cloud';
+
+    $: if (activeToggle && typeof window !== 'undefined') {
+        const params = new URLSearchParams(window.location.search);
+        params.set('type', activeToggle);
+        goto(`?${params.toString()}`, { replaceState: true });
```

Comment:
```suggestion
        goto(`?${params.toString()}`, { replaceState: true, noScroll: true });
```
This would avoid a jump to the top of the screen if there was a bit of an initial scroll. Wdyt?